src/fluid.cpp

```diff
@@ -362,8 +362,8 @@ CDynamicAddress CFluid::GetAddressFromDigestSignature(std::string digestSignatur
     CPubKey pubkey;
 
     if (!pubkey.RecoverCompact(ss.GetHash(), vchSig)) {
-        LogPrintf("GenericVerifyInstruction(): Public Key Recovery Failed! Hash: %s\n", ss.GetHash().ToString());
-        return false;
```

Comment:
This is temporary to allow building. This is from merging Amirs work, will await the fix for this from him.